po/pl.po

```diff
@@ -1065,14 +1066,13 @@ msgstr "Błąd: użycie filtra nie powiodło się: %s"
 #, c-format
 msgid "1 day ago"
 msgid_plural "%u days ago"
-msgstr[0] ""
-msgstr[1] ""
-msgstr[2] ""
+msgstr[0] "1 dzień temu"
+msgstr[1] "%u dni temu"
+msgstr[2] "%u dni temu"
```

Comment:
I don't know Polish, but it's a bit suspicious that translations for the two plural forms are identical. Not saying this is an error, just want you to take another look and make sure this *is* correct.

Reply:
I will double check, but AFAIK
1 day ==1 dzień
>=2 days == >=2 dni

Reply:
I will double check, but AFAIK
1 day ==1 dzień
>=2 days == >=2 dni

Reply:
Gettext seems to believe it's a bit more complex: https://github.com/newsboat/newsboat/blob/d640bcdc62f4ca3f6539fd86e9d097e741623e81/po/pl.po#L17-L18

It looks like numbers 12, 13, 14, 112, 113, 114 etc. would use the first translation, while others would use the second one.

Reply:
Ah, no, I'm reading the code wrong. Here's [a program that generates examples](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=a87622b7966fb806aa93ea7c4679022e) (press Ctrl+Enter to run it).

Reply:
Interesting, indeed it looks like it is common practice for pl translations e.g: https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals

But it case of days/dni I cannot see a difference :D Will try to dig some more in the evening.

Reply:
Oh, that link explains it:

> Even though the last 2 plural forms of "day" are the same, both are still needed, because there needs to be 3 plural forms for each word.

Disregard this comment then.